sysutils/just/Portfile

```diff
@@ -2,28 +2,95 @@
 
 PortSystem          1.0
 PortGroup           github 1.0
+PortGroup           cargo 1.0
 
-github.setup        casey just 0.2.0
+github.setup        casey just v0.4.4
 github.tarball_from releases
 
 categories          sysutils
 platforms           darwin
+maintainers         {@casey rodarmor.com:casey}
 license             permissive
+homepage            https://github.com/casey/just
```

Comment:
The `homepage` is already set to this value by the `github.setup` command. So this line should be removed.

Reply:
fixed